ide/static/js/netLayout.js

```diff
@@ -69,7 +68,7 @@ let i = null, layerId = null, parentId =  null, inputLength = null, outputLength
 
 // finding the input layers to start DFS
 Object.keys(net).forEach(layerId => {
-  if (net[layerId].info.type === 'Data' || net[layerId].info.type === 'Input') {
+  if (net[layerId].info.type === 'Data' || net[layerId].info.type === 'Input' || net[layerId].info.type === 'HDF5Data' || net[layerId].info.type === 'Reshape') {
     stack.push(layerId);
```

Comment:
It won't be the first layer, but as in this network it is the first layer after the Data layers.
When I tried to run the prototxt file without adding Reshape as an input it failed, I assumed this is because Reshape is a special layer like Inputs.